android/src/main/java/io/ably/flutter/plugin/AblyFlutter.java

```diff
@@ -39,22 +39,6 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBindin
         setupChannels(flutterPluginBinding.getBinaryMessenger(), applicationContext);
     }
 
-    // This static function is optional and equivalent to onAttachedToEngine. It supports the old
-    // pre-Flutter-1.12 Android projects. You are encouraged to continue supporting
-    // plugin registration via this function while apps migrate to use the new Android APIs
-    // post-flutter-1.12 via https://flutter.dev/go/android-project-migration.
-    //
-    // It is encouraged to share logic between onAttachedToEngine and registerWith to keep
-    // them functionally equivalent. Only one of onAttachedToEngine or registerWith will be called
-    // depending on the user's project. onAttachedToEngine or registerWith must both be defined
-    // in the same class.
-    public static void registerWith(Registrar registrar) {
-        Context applicationContext = registrar.context().getApplicationContext();
-        final AblyFlutter ably = new AblyFlutter();
-        registrar.addNewIntentListener(ably);
-        ably.setupChannels(registrar.messenger(), applicationContext);
-    }
-
```

Comment:
Can you / should you inform users about this adding some content to Readme?

Reply:
Good question, but in our case, I don't think it's necessary to disclose this anywhere.
* This was deprecated in Flutter version 1.22, as far as I can tell, our plugin targets Flutter > 2.0, so I highly doubt it's even possible to use this in current configuration.
* Wider context on this is provided in official documentation [here](https://docs.flutter.dev/release/breaking-changes/android-v1-embedding-create-deprecation#context), but in general, this is a very old solution that should have been removed quite a while ago. Following Flutter release notes: `As of Q2 2020, only 26% of applications used the v1 embeddings`, so I think we can safely assume that this shouldn't affect anyone

So I think disclosing this may be just unwanted noise in the docs